<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventario</class>
 <widget class="QMainWindow" name="Inventario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGestionarRepuestos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestionar Repuestos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGestionarProveedores">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestionar Proveedores</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
